docs/src/gs/create-workload-cluster.md

```diff
@@ -8,15 +8,19 @@ Choose one of the available templates for to create your workload clusters from
+| Parameter                                 | Default Value       | Description                                                                                                                                                                                                                                                                                                                                                                                                    |
+|-------------------------------------------|---------------------|----------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------|
+| `OCI_COMPARTMENT_ID`                      |                     | The OCID of the compartment where the OCI resources are to be created                                                                                                                                                                                                                                                                                                                                          |
+| `OCI_IMAGE_ID`                            |                     | The OCID of the Compute Image (Oracle Linux or <br/>Ubuntu) with which to create the Kubernetes nodes. This same image is used for both the control plane and the worker nodes.                                                                                                                                                                                                                                |
+| `OCI_CONTROL_PLANE_SHAPE`                 | VM.Standard.E4.Flex | The shape of the Kubernetes nodes                                                                                                                                                                                                                                                                                                                                                                              |
+| `OCI_CONTROL_PLANE_SHAPE_MEMORY_IN_GBS`   |                     | The amount of memory in GBs to be allocated to the control plane instances. If not provided, the memory is automatically computed by compute API.                                                                                                                                                                                                                                                              |
```

Comment:
I suggest expanding GBs to gigabytes unless you actually meant gibibytes. I would also replace "compute API" with something like "the memory is automatically allocated at 8GB per OCPU."

Reply:
> "the memory is automatically allocated at 8GB per OCPU."

but that isn't true. BM Machines have a much different GB to OPU allocation I believe.

Reply:
- [ ] After talking: "Memory is automatically allocated based on the chosen shape and OCPU count"